Refinements in section 1 (#17)
ypriverol/os-proteomics: manuscript.md

Unlike "closed-source" or "proprietary" software, where the code remains private, OSS provides transparency by design, which is essential for fostering trust, collaboration, and progress in science.
However, the open- or closed-source label simply addresses code visibility; it is the software license that determines who can use, modify, or distribute the software and under what conditions.

In proteomics - and bioinformatics in general - multiple misconceptions exist about open/closed source software: 

- Closed-source software with "academic" licenses are equivalent to open-source. This is not true, as academic licenses only refer to free-for-academic-use, but the source code is not necessarily open, shareable or can be modified. Even the term "academic" is not well-defined, as it can refer to a wide range of institutions and organizations. To simplify this complexity, we can define OSS as any software that uses an OSI-approved license (https://opensource.org/licenses). 

> - Open-source means the code accessible. This is not true, Open-source software does not only mean that the code is openly available, but also that it is allowed to be modified and shared.

Reviewer: ```suggestion
- Open-source means that only the code accessible. This is not true, open-source software does not only mean that the code is openly available, but also that it is allowed to be modified and shared.
```

> In addition to all these misconceptions and complexity, journals mandate code availability as a publication requirement, which has prompted multiple bad practices from software developers and bioinformaticians aiming to futfill the journal's requirements. 

Reviewer: ```suggestion
In addition to all these misconceptions and complexity, many journals mandate code availability as part of publishing, which has prompted multiple bad practices from software developers and bioinformaticians aiming to fulfill these requirements. 
```

> - Researchers upload closed-source software to platforms like GitHub, giving an impression of openness with features such as issue tracking, while the actual source code remains inaccessible. Although often well-intentioned, this practice can mislead other scientists and, in our view, should be discouraged. 

Reviewer 2: Although not against the terms of GH. I think the sentence as-is is fine.

> - Software is deposited in GitHub as open-source during the submission of the manuscript, but after publication; software licenses in GitHub repositories are changed over time, and repositories themselves can be deleted or made private, complicating efforts to ensure long-term accessibility and reliability.

Reviewer: ```suggestion
- Software is deposited in GitHub as open-source during the submission of the manuscript, but after publication; software licenses in GitHub repositories are changed, repositories deleted or made private, complicating efforts to ensure long-term accessibility.
```

Reviewer 2: Maybe something like this?

"Software deposited on GitHub as open-source during the submission can change its license after publication of the manuscript, or even worse, repositories get deleted or made private. These steps complicate efforts to ensure long-term accessibility to the code."

Reviewer 3: This is a pretty complex sentence, I suggest splitting it up in two.

> Knowing all of these misconceptions and complexity, we want to make clear for the community, users, developers, readers of manuscripts, and reviewers what is open-source in proteomics, what is not, and how to make it clear for everyone: 

Reviewer: ```suggestion
Given the many misconceptions and complexities surrounding open-source in proteomics, we aim to clarify for the entire community - users, developers, manuscript readers, and reviewers - what constitutes open-source, what does not, and how to transparently communicate the state of own software: 
```

- **Source Code Availability**: The source code is publicly accessible, allowing anyone to view, download, and examine the code’s details (https://opensource.org/osd).

- **OSI-Approved License**: The software must use an OSI-approved license, which specifies rights to freely use, modify, and distribute the software, regardless of its application or environment (https://opensource.org/licenses).

> - **Freedom to Modify and Distribute**: Open-source software allows users not only to access the code but also to modify it and share these modifications, encouraging innovation and collaboration.

Reviewer 2: Could clarify some confusion about open source software vs source-available but no modifications can be me (non-OSI):

"Open-source software, in contrast to source-available software, allows users ..."

### Shared knowledge pushes the field forward

Open-source software (OSS) fosters a collaborative ecosystem where researchers across institutions can freely contribute, refine, and extend tools, accelerating scientific progress. 
Unlike proprietary software that confines advancements to specific labs or companies, OSS allows researchers to build on each other’s work without duplicating efforts, promoting efficient resource use and transforming individual achievements into collective gains. 
By democratizing access to high-quality tools, OSS enables scientists from resource-limited settings to leverage the same technology as those in well-funded institutions, fostering a diverse global community that drives innovation forward.

> Proteomics has greatly benefited from this open-source approach.

Reviewer 3: This section is very nice now. 👍
